tests/installation/release_notes_from_url.pm

```diff
@@ -19,7 +19,13 @@ sub run {
     enter_cmd "zgrep -oh \"Got release notes.*\" /var/log/YaST2/y2log*";
     assert_screen [qw(got-releasenotes-RPM got-releasenotes-URL)];
     unless (match_has_tag 'got-releasenotes-URL') {
-        record_soft_failure('bsc#1190711 - Release notes source does NOT match expectations or not found in YaST logs, expected source: URL');
+        my $not_public = script_run('grep -rE "GMC|RC" /README.BETA');
+        unless (get_var('BETA') || $not_public) {
```

Comment:
In case we have a GM build, which means that README.BETA will not exist and then the script_run command will return 1 because of the grep failure, the die will not be executed, therefore I added the BETA check.

Reply:
we don't have that build, we only test until release candidate (at least in installations).

Reply:
The last build 102.1 for 15SP5 was not BETA, that's why I took this under consideration

Reply:
hmm.. interesting, this was the GMC... so no README.BETA for GMC...

Reply:
SO I removed the GMC  from the grep regex and now only ise the BETA var to check if it's GMC build